mypy/test/testcheck.py

```diff
@@ -187,8 +187,11 @@ def find_module_files(self) -> Dict[str, str]:
             del dnparts[0]
             for file in files:
                 if file.endswith('.py'):
-                    base, ext = os.path.splitext(file)
-                    id = '.'.join(dnparts + [base])
+                    if file == "__init__.py":
```

Comment:
@ddfisher -- I submitted a pull request with the comment here: https://github.com/python/mypy/pull/1844